src/utils/filterutils.tsx

```diff
@@ -119,3 +124,25 @@ export function getSelectedNumericalFacetFields(searchActions: SearchActions): S
   ) ?? [];
   return new Set(selectedNumericalFacets.map(f => f.fieldId));
 }
+
+/**
+ * Goes through the provided static filters and returns all that are field value
+ * filters.
+ *
+ * @param staticFilters - The list of {@link SelectableStaticFilter}s to search through
+ * @returns All filters that are {@link SelectableFieldValueFilter}s
+ */
+export function getSelectableFieldValueFilters(
+  staticFilters: SelectableStaticFilter[]
+): SelectableFieldValueFilter[] {
+  return staticFilters.reduce((fieldValueFilters: SelectableFieldValueFilter[], s) => {
```

Comment:
Can we use `filter` instead of `reduce` here?

Reply:
I think `filter` would only work if we wanted to return a list of type `SelectableStaticFilter[]`. since we're parsing the static filters and returning a different type, especially one that's not one of the `StaticFilter` union types, I used `reduce`

Reply:
Does .filter combined with .map work? I don't think there's anything wrong with reduce it's just slightly harder to understand (because you can do more complicated things with it)

Reply:
splitting the `reduce` into a `filter` and then a `map` would have type errors because `filter` still returns a type of `SelectableStaticFilter[]`, and `map` wouldn't know that it technically only has field value static filters in it after the filtering. so we would have to repeat the type checking in the `map` to access the `fieldId`, `matcher`, and `value` properties. but, we would still have to handle the case where TS thinks it might not be a field value static filter. essentially, we would have to either do a manual assertion, or allow `undefined` to be returned in the array like this:

<img width="617" alt="Screen Shot 2022-08-17 at 2 05 10 PM" src="https://user-images.githubusercontent.com/88398086/185210983-e26b5172-601a-4ce2-a1e2-e567d599675e.png">

Reply:
hmm we could do this if you like? but I see your point, either works for me!
```ts

  return staticFilters.map(s => {
    const { filter: { kind, ...filterFields }, ...displayFields } = s;
    if (kind === 'fieldValue') {
      return {
        ...displayFields,
        ...filterFields
      };
    }
    return undefined;
  }).filter((s): s is SelectableFieldValueFilter => !!s);
```

Reply:
that works, updated!